extras/rya.export/export.accumulo/src/main/java/org/apache/rya/export/accumulo/AccumuloParentMetadataRepository.java

```diff
@@ -29,15 +61,210 @@
  * <li>Timestamp used as the lower cutoff for the export</li>
  */
 public class AccumuloParentMetadataRepository implements ParentMetadataRepository {
-    private MergeParentMetadata metadata;
+    private static final Logger log = Logger.getLogger(AccumuloParentMetadataRepository.class);
+
+    /**
+     * The Row ID of all {@link MergeParentMetadata} entries that are stored in Accumulo.
+     */
+    private static final Text MERGE_PARENT_METADATA_ROW_ID = new Text("MergeParentMetadata");
+
+    /**
+     * The Column Family for all MergeParentMetadata entries.
+     */
+    private static final Text MERGE_PARENT_METADATA_FAMILY = new Text("mergeParentMetadata");
+
+    /**
+     * The Column Qualifier for the Rya instance name.
+     */
+    private static final Text MERGE_PARENT_METADATA_RYA_INSTANCE_NAME = new Text("ryaInstanceName");
+
+    /**
+     * The Column Qualifier for the timestamp.
+     */
+    private static final Text MERGE_PARENT_METADATA_TIMESTAMP = new Text("timestamp");
+
+    /**
+     * The Column Qualifier for the copy tool input time.
+     */
+    private static final Text MERGE_PARENT_METADATA_COPY_TOOL_INPUT_TIME = new Text("copyToolInputTime");
+
+    /**
+     * The Column Qualifier for the copy tool run time.
+     */
+    private static final Text MERGE_PARENT_METADATA_COPY_TOOL_RUN_TIME = new Text("copyToolRunTime");
+
+    /**
+     * The Column Qualifier for the parent time offset.
+     */
+    private static final Text MERGE_PARENT_METADATA_PARENT_TIME_OFFSET = new Text("parentTimeOffset");
+
+    // Lexicoders used to read/write MergeParentMetadata to/from Accumulo.
+    private static final LongLexicoder LONG_LEXICODER = new LongLexicoder();
+    private static final StringLexicoder STRING_LEXICODER = new StringLexicoder();
+    private static final DateLexicoder DATE_LEXICODER = new DateLexicoder();
+
+    private static final String BASE_MERGE_METADATA_TABLE_NAME = "merge_metadata";
+
+    private final AccumuloRyaDAO accumuloRyaDao;
+    private final Connector connector;
+    private final String tablePrefix;
+    private final String mergeParentMetadataTableName;
+
+    /**
+     * Creates a new instance of {@link AccumuloParentMetadataRepository}.
+     * @param accumuloRyaDao the {@link AccumuloRyaDAO}. (not {@code null})
+     */
+    public AccumuloParentMetadataRepository(final AccumuloRyaDAO accumuloRyaDao) {
+        this.accumuloRyaDao = checkNotNull(accumuloRyaDao);
+        connector = accumuloRyaDao.getConnector();
+        tablePrefix = accumuloRyaDao.getConf().getTablePrefix();
+        mergeParentMetadataTableName = tablePrefix + BASE_MERGE_METADATA_TABLE_NAME;
+    }
 
     @Override
     public MergeParentMetadata get() {
-        return metadata;
+        checkNotNull(connector);
```

Comment:
use an actual if() null check, then if it is null, throw an exception about not being instantiated or some accumulo based exception